pages/common/tailscale.md

```diff
@@ -32,6 +32,6 @@
 
 `tailscale bugreport`
 
-- Display help for a subcommand:
+- Display debug helper commands (undocumented):
```

Comment:
it's documented, it has a help page in the terminal.

Reply:
It does but only when one does `tailscale debug --help`. The normal `tailscale --help` does not show it.
IMO, it happens to be pretty useful :)

Reply:
If as you say below the `debug` subcommand is unstable, it does not seem wise to document it here.

Reply:
In my opinion, the reason for adding a `debug` example instead of a page is just to provide the user a hint that it exists :)
It in my opinion is unstable to warrant a separate page but useful enough for debug purposes :)
@sbrl